packages/vue/src/components/basic/SelectedFilters.jsx

```diff
@@ -67,7 +69,23 @@ const SelectedFilters = {
 			this.$emit('clear', component, value);
 		},
 		clearValues() {
-			this.clearValuesAction();
+			const { resetToDefault, componentProps, selectedValues } = this;
+			if (resetToDefault) {
+				Object.keys(selectedValues).map(component => {
+					if (
+						componentProps[component].defaultValue
```

Comment:
@mohdashraf010897 Add a `null` check before reading value. You can fix it at all the instances. Component shouldn't break if something is `undefined`.